### PR TITLE
releases: add tidb v4.0.11 release notes 

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

add tidb v4.0.11 release notes
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4761
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-4.0.11.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# TiDB 4.0.11 Release Notes
+
+发版日期：2021 年 2 月 4 日
+
+TiDB 版本：4.0.11
+
+## 兼容性更改
+
+## 新功能
+
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:14]:
```suggestion

+ TiFlash

    - 增加排队处理 Coprocessor 任务的线程池以减小 OOM 几率, 随之增加两项配置项 `cop_pool_size` 与 `batch_cop_pool_size`, 默认为 `物理核数 * 2`. [#1312](https://github.com/pingcap/tics/pull/1312)

```

File: releases/release-4.0.11.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## 改进提升
+
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:16]:
```suggestion

+ TiFlash
    - 优化 date_format 函数的性能 [#1339](https://github.com/pingcap/tics/pull/1339)
    - 优化处理 ingest SST 时的内存开销

```

File: releases/release-4.0.11.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+## Bug 修复
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:17]:
```suggestion
## Bug 修复

+ TiFlash

    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)
```

File: releases/release-4.0.11.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
++ TiFlash
+
+    - 优化 date_format 函数的性能 [#1339](https://github.com/pingcap/tics/pull/1339)
+    - 优化处理 ingest SST 时的内存开销
+
+## Bug 修复
+
++ TiFlash
+
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:29]:
```suggestion

    - 修正 decimal 类型计算 `min/max`的结果错误
```

File: releases/release-4.0.11.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    - 修正 decimal 类型计算 `min/max`的结果错误
+    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
+    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)
```
Comment (overvenus) [inline releases/release-4.0.11.md:32]:
```suggestion
    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)

# TiCDC

## Improvements

* Add version in capture info and changefeed info [#1342](https://github.com/pingcap/ticdc/pull/1342)

## Bug Fixes

* Fix a bug that cdc server could exit unexpected when meeting ErrTaskStatusNotExists and the capture session is disconnected at the same time [#1240](https://github.com/pingcap/ticdc/pull/1240)
* Fix the old-value switch of a changefeed could be affected by another changefeed [#1347](https://github.com/pingcap/ticdc/pull/1347)
* Fix a bug that TiCDC server could hang when processing a new changefeed with invalid sort-engine parameter [#1309](https://github.com/pingcap/ticdc/pull/1309)
* fix debug info panic on none owner node [#1349](https://github.com/pingcap/ticdc/pull/1349)
* Fix metric `ticdc_processor_num_of_tables` and `ticdc_processor_table_resolved_ts` are properly updated when processor removes a table or the processor itself stops [#1351](https://github.com/pingcap/ticdc/pull/1351)
* Fix potential data loss if a processor crashes when starting up a table [#1363](https://github.com/pingcap/ticdc/pull/1363)
* Fix a bug in the owner that could lead to abnormal CDC server exits during table migrations [#1352](https://github.com/pingcap/ticdc/pull/1352)
* Fix a bug TiCDC does not fail in time after service safepoint is lost [#1367](https://github.com/pingcap/ticdc/pull/1367)
* Fix a bug that kv client may skip to recreate event feed receiving routine by accident [#1336](https://github.com/pingcap/ticdc/pull/1336)
* Fix a bug that atomicity of transactions is broken in the downstream [#1375](https://github.com/pingcap/ticdc/pull/1375)

# TiDB-binlog

None

# BR

## Bug Fixes

* Fix the issue that missing file size in SSTMeta might cause TiKV to generate a big region [#702](https://github.com/pingcap/br/pull/702)
* Fix the issue that br restores table auto id even if the table does not have one [#720](https://github.com/pingcap/br/pull/720)

# Dumpling

None

# TiDB-Lightning

## Improvements

* Improve import performance by creating tables in parallel [#502](https://github.com/pingcap/tidb-lightning/pull/502)
* Improve import performance by skipping split regions if engine total size is smaller than region size [#524](https://github.com/pingcap/tidb-lightning/pull/524)
* Add importing progress and optimize the accuracy of restore progress [#506](https://github.com/pingcap/tidb-lightning/pull/506)

## Bug Fixes

* Fix the bug that TiDB-Lightning will trim all the empty sep when trim-last-sep is true, which causes "column count mismatch" in tidb backend [#535](https://github.com/pingcap/tidb-lightning/pull/535)
* Fix the bug that tidb backend will panics if source file columns are more than target table columns [#528](https://github.com/pingcap/tidb-lightning/pull/528)
* Fix the bug that TiKV may panics if TiDB-Lightning retry ingest with retry write [#554](https://github.com/pingcap/tidb-lightning/pull/554)

```

Comment (overvenus) [inline releases/release-4.0.11.md:32]:
```suggestion
    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)

# TiCDC

## Improvements

* 在 Capture 元信息添加版本信息和在 Changefeed 元信息中创建该 Changefeed 的 CLI 版本 [#1342](https://github.com/pingcap/ticdc/pull/1342)

## Bug Fixes

* 修复 TiCDC 服务在同时发生 ErrTaskStatusNotExists 和 Capture 会话关闭的情况下的非预期的退出 [#1240](https://github.com/pingcap/ticdc/pull/1240)
* 修复不用 Changefeed 之间不同 old-value 设置的互相影响问题 [#1347](https://github.com/pingcap/ticdc/pull/1347)
* 修复 TiCDC 服务在遇见错误 sort-engine 参数时卡住的问题 [#1309](https://github.com/pingcap/ticdc/pull/1309)
* 修复在非 Owner 节点上获取 debug 信息退出的问题 [#1349](https://github.com/pingcap/ticdc/pull/1349)
* 修复 `ticdc_processor_num_of_tables` and `ticdc_processor_table_resolved_ts` 两个监控指标在增删数据表时没有被正确更新的问题 [#1351](https://github.com/pingcap/ticdc/pull/1351)
* 修复 Processor 在添加同步数据表时退出而造成的潜在的数据丢失问题 [#1363](https://github.com/pingcap/ticdc/pull/1363)
* 修复 Owner 在数据表迁移期间造成非正常的状态的 TiCDC 服务的退出问题 [#1352](https://github.com/pingcap/ticdc/pull/1352)
* 修复 TiCDC 服务在丢失 service gc safepoint 是没有及时退出的问题 [#1367](https://github.com/pingcap/ticdc/pull/1367)
* 修复 kv client 潜在的跳过创建 event feed 的问题 [#1336](https://github.com/pingcap/ticdc/pull/1336)
* 修复同步事务到下游时事务原子性被破坏的问题 [#1375](https://github.com/pingcap/ticdc/pull/1375)

# TiDB-binlog

None

# BR

## Bug Fixes

* 修复恢复备份后可能产生大 region 的问题 [#702](https://github.com/pingcap/br/pull/702)
* 修复在没有 auto id 的数据表上恢复 auto id 的问题 [#720](https://github.com/pingcap/br/pull/720)

# Dumpling

None

# TiDB-Lightning

## Improvements

* 并行创建数据表以改进导入速度 [#502](https://github.com/pingcap/tidb-lightning/pull/502)
* 跳过分裂小 region 以改进导入速度 [#524](https://github.com/pingcap/tidb-lightning/pull/524)
* 添加导入进度条并改进导入进度精度 [#506](https://github.com/pingcap/tidb-lightning/pull/506)

## Bug Fixes

* 修复使用 TiDB backend 时可能触发的 "column count mismatch" 问题 [#535](https://github.com/pingcap/tidb-lightning/pull/535)
* 修复 TiDB backend 在导入数据源 column 个数和数据表 column 个数不匹配时非预期退出的问题 [#528](https://github.com/pingcap/tidb-lightning/pull/528)
* 修复导入期间 TiKV 可能发生的非预期退出的问题 [#554](https://github.com/pingcap/tidb-lightning/pull/554)

```

File: releases/release-4.0.11.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## 改进提升
+
++ TiFlash
+
+    - 优化 date_format 函数的性能 [#1339](https://github.com/pingcap/tics/pull/1339)
+    - 优化处理 ingest SST 时的内存开销
+
+## Bug 修复
+
```
Comment (NingLin-P) [inline releases/release-4.0.11.md:27]:
```suggestion
+ TiKV

    - 修复当设置 PROST=1 时构建 TiKV 失败的问题 [#9604](https://github.com/tikv/tikv/pull/9604)
    - 修复不匹配的内存诊断信息 [#9589](https://github.com/tikv/tikv/pull/9589)
    - 修复在恢复 rawkv 数据时部分 key range 的 end key 的包含性问题 [#9583](https://github.com/tikv/tikv/pull/9583)
    - 修复当 CDC 增量扫数据时读取一个被回滚的事务的某个 key 的旧值时 TiKV 可能会 panic 的问题 [#9569](https://github.com/tikv/tikv/pull/9569)
    - 修复使用不同配置的连接拉取同一个 region 的变更时旧值配置不匹配的问题  [#9565](https://github.com/tikv/tikv/pull/9565)
    - 修复 TiVK 运行在网络接口缺少 MAC 地址的设备上会崩溃的问题（自 v4.0.9 引入）[#9516](https://github.com/tikv/tikv/pull/9516)
    - 修复 TiKV 在备份大 region 时会 OOM 的问题 [#9448](https://github.com/tikv/tikv/pull/9448)
    - 修复 `region-split-check-diff` 无法配置的问题 [#9530](https://github.com/tikv/tikv/pull/9530)
    - 修复系统时间回退时 TiKV 会 panic 的问题 [#9542](https://github.com/tikv/tikv/pull/9542)
```

File: releases/release-4.0.11.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    - 增加排队处理 Coprocessor 任务的线程池以减小 OOM 几率, 随之增加两项配置项 `cop_pool_size` 与 `batch_cop_pool_size`, 默认为 `物理核数 * 2`. [#1312](https://github.com/pingcap/tics/pull/1312)
+
+## 改进提升
+
++ TiFlash
+
+    - 优化 date_format 函数的性能 [#1339](https://github.com/pingcap/tics/pull/1339)
+    - 优化处理 ingest SST 时的内存开销
+
```
Comment (nolouch) [inline releases/release-4.0.11.md:25]: + PD
    - Grafana dashboards 支持监控多个集群 [#3398](https://github.com/tikv/tikv/pull/3398)

File: releases/release-4.0.11.md (hunk 6 above)
Comment (nolouch) [inline releases/release-4.0.11.md:27]: + PD
    - 修复成员健康的监控显示不正确的问题 [#3368](https://github.com/pingcap/pd/pull/3368)
    - 检查禁止有副本的不正常 tombstone store 被清除 [#3352](https://github.com/pingcap/pd/pull/3352)
    - 修复 store limit 无法持久化的问题  [#3403](https://github.com/pingcap/pd/pull/3403)

File: releases/release-4.0.11.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+# TiDB 4.0.11 Release Notes
+
+发版日期：2021 年 2 月 4 日
+
+TiDB 版本：4.0.11
+
+## 兼容性更改
+
+## 新功能
+
```
Comment (qw4990) [inline releases/release-4.0.11.md:14]:
```suggestion
+ TiDB

    - 支持 `uft8_unicode_ci` 和 `utf8mb4_unicode_ci` collation [#22558](https://github.com/pingcap/tidb/pull/22558)
```

File: releases/release-4.0.11.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    - 支持 `utf8mb4_unicode_ci` 函数 [#9577](https://github.com/tikv/tikv/pull/9577)
+    - 支持 `cast_year_as_time` 函数 [#9299](https://github.com/tikv/tikv/pull/9299)
+
++ TiFlash
+
+    - 增加排队处理 Coprocessor 任务的线程池以减小 OOM 几率, 随之增加两项配置项 `cop_pool_size` 与 `batch_cop_pool_size`, 默认为 `物理核数 * 2`. [#1312](https://github.com/pingcap/tics/pull/1312)
+
+## 改进提升
+
```
Comment (qw4990) [inline releases/release-4.0.11.md:25]:
```suggestion
+ TiDB
    - 重排由 `outer join` 简化的 `inner join` 顺序 [#22402](https://github.com/pingcap/tidb/pull/22402)
    - Grafana 面板支持多集群 [#22534](https://github.com/pingcap/tidb/pull/22534)
    - 慢查询监控被区分为 internal 和 general 两类 [#22405](https://github.com/pingcap/tidb/pull/22405)
    - 增加 `utf8_unicode_ci` 和 `utf8mb4_unicode_ci` collatin 接口 [#22099](https://github.com/pingcap/tidb/pull/22099)
```

File: releases/release-4.0.11.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+        - 在 Capture 元信息添加版本信息和在 Changefeed 元信息中创建该 Changefeed 的 CLI 版本 [#1342](https://github.com/pingcap/ticdc/pull/1342)
+
+    + TiDB Lightning
+
+        * 并行创建数据表以改进导入速度 [#502](https://github.com/pingcap/tidb-lightning/pull/502)
+        * 跳过分裂小 region 以改进导入速度 [#524](https://github.com/pingcap/tidb-lightning/pull/524)
+        * 添加导入进度条并改进导入进度精度 [#506](https://github.com/pingcap/tidb-lightning/pull/506)
+
+## Bug 修复
+
```
Comment (qw4990) [inline releases/release-4.0.11.md:54]:
```suggestion
    - 修复异常的 `unicode_ci` 常数传递 [#22614](https://github.com/pingcap/tidb/pull/22614)
    - 修复可能导致 collation 和 coercibility 错误的问题 [#22602](https://github.com/pingcap/tidb/pull/22602)
    - 修复可能导致错误 collation 结果的问题 [#22599](https://github.com/pingcap/tidb/pull/22599)
    - 修复不同 collation 的常数替换问题 [#22582](https://github.com/pingcap/tidb/pull/22582)
    - 修复 `like` 函数使用 collation 时可能返回错误结果的问题 [#22531](https://github.com/pingcap/tidb/pull/22531)
    - 修复 `least` 和 `greatest` 函数 duration 类型推导错误问题 [#22580](https://github.com/pingcap/tidb/pull/22580)
    - 修复 `like` 函数处理 `_` 宽字符后加 `%` 出错的问题 [#22575](https://github.com/pingcap/tidb/pull/22575)
    - 修复比较函数 `least` 和 `greatest` 类型推导错误的问题 [#22562](https://github.com/pingcap/tidb/pull/22562)
    - 修复使用 `like` 函数处理 Unicode 字符串错误的问题 [#22529](https://github.com/pingcap/tidb/pull/22529)
    - 修复点查请求无法取得 `@@tidb_snapshot` 变量中快照的问题 [#22527](https://github.com/pingcap/tidb/pull/22527)
    - 修复生成多个 join 相关 hint 可能 panic 的问题 [#22518](https://github.com/pingcap/tidb/pull/22518)
    - 修复转换字符串为 BIT 类型不准确的问题 [#22420](https://github.com/pingcap/tidb/pull/22420)
    - 修复插入 `tidb_rowid` 列时出现的 'index out of range' 报错问题 [#22359](https://github.com/pingcap/tidb/pull/22359)
    - 修复缓存计划被错误的使用问题 [#22353](https://github.com/pingcap/tidb/pull/22353)
    - 修复 `WEIGHT_STRING` 函数处理过长字符串 panic 的问题 [#22332](https://github.com/pingcap/tidb/pull/22332)
    - 禁止参数数量不合法时使用生成列 [#22174](https://github.com/pingcap/tidb/pull/22174)
    - 在构造执行计划前正确的设置进程执行信息 [#22148](https://github.com/pingcap/tidb/pull/22148)
    - 修复 IndexLookUp 执行统计不准的问题 [#22136](https://github.com/pingcap/tidb/pull/22136)
    - 容器部署时为内存使用信息增加 cache [#22116](https://github.com/pingcap/tidb/pull/22116)
    - 修复解码执行计划错误的问题 [#22022](https://github.com/pingcap/tidb/pull/22022)
    - 使用错误的窗口函数说明时报错 [#21976](https://github.com/pingcap/tidb/pull/21976)
    - 使用 prepare 语句嵌套 execute, deallocate, prepare 时报错 [#21972](https://github.com/pingcap/tidb/pull/21972)
    - 修复使用 `insert ignore` 到不存在的 partition 不报错的问题 [#21971](https://github.com/pingcap/tidb/pull/21971)
    - 统一 explain 和 slow log 中的执行计划编码 [#21964](https://github.com/pingcap/tidb/pull/21964)
    - 修复聚合算子下 join 出现未知列的问题 [#21957](https://github.com/pingcap/tidb/pull/21957)
    - 修复 ceiling 函数中类型推导错误的问题 [#21936](https://github.com/pingcap/tidb/pull/21936)
    - 修复 double 列忽略精度的问题 [#21916](https://github.com/pingcap/tidb/pull/21916)
    - 修复关联聚合在子查询中被计算的问题 [#21877](https://github.com/pingcap/tidb/pull/21877)
    - 当 JSON 数据长度超过 65536 时报错 [#21870](https://github.com/pingcap/tidb/pull/21870)
    - 修复 `dyname` 函数和 MySQL 不兼容的问题 [#21850](https://github.com/pingcap/tidb/pull/21850)
    - 修复输入数据过长时 `to_base64` 函数返回 NULL 的问题 [#21813](https://github.com/pingcap/tidb/pull/21813)
    - 修复在子查询中比较多个字段失败的问题 [#21808](https://github.com/pingcap/tidb/pull/21808)
    - 修复 JSON 中比较浮点数的问题 [#21785](https://github.com/pingcap/tidb/pull/21785)
    - 修复 JSON 类型比较的问题 [#21718](https://github.com/pingcap/tidb/pull/21718)
    - 修复 cast 函数 coercibility 值设置错误的问题 [#21714](https://github.com/pingcap/tidb/pull/21714)
    - 修复使用 IF 函数时可能出现 panic 的问题 [#21711](https://github.com/pingcap/tidb/pull/21711)
    - 修复 json 搜索返回 null 和 MySQL 不兼容的问题 [#21700](https://github.com/pingcap/tidb/pull/21700)
    - 修复 ORDER BY 和 HAVING 字句检查 only_full_group_by 模式的问题 [#21697](https://github.com/pingcap/tidb/pull/21697)
    - 修复 day/time 单位和 MySQL 不兼容的问题 [#21676](https://github.com/pingcap/tidb/pull/21676)
    - 修复 LEAD 和 LAG 函数默认值类型问题 [#21665](https://github.com/pingcap/tidb/pull/21665)
    - `LOAD DATA` 时检测以保证只能往基础表中导入数据 [#21638](https://github.com/pingcap/tidb/pull/21638)
    - 修复 addtime 和 subtime 函数处理非法参数的问题 [#21635](https://github.com/pingcap/tidb/pull/21635)

```

File: releases/release-4.0.11.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+    - 重排由 `outer join` 简化的 `inner join` 顺序 [#22402](https://github.com/pingcap/tidb/pull/22402)
+    - Grafana 面板支持多集群 [#22534](https://github.com/pingcap/tidb/pull/22534)
+    - 为多语句问题提供替代解决方案 [#22468](https://github.com/pingcap/tidb/pull/22468)
+    - 将慢查询监控区分为 `internal` 和 `general` 两类 [#22405](https://github.com/pingcap/tidb/pull/22405)
+    - 为 `utf8_unicode_ci` 和 `utf8mb4_unicode_ci` 排序规则增加接口 [#22099](https://github.com/pingcap/tidb/pull/22099)
+
++ TiKV
+
+    - 为 DBasS 添加 server 信息的监控指标 [#9591](https://github.com/tikv/tikv/pull/9591)
```
Comment (alex-quan-001) [inline releases/release-4.0.11.md:38]: typo：DBasS should be DBaaS

File: releases/release-4.0.11.md [hunk 12]
```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 4.0.11 Release Notes
+---
+
+# TiDB 4.0.11 Release Notes
+
+发版日期：2021 年 2 月 4 日
```
Comment (alex-quan-001) [inline releases/release-4.0.11.md:7]: TBD, not yet determined

**Review by alex-quan-001 — COMMENTED** [review]
a typo

File: releases/release-4.0.11.md (hunk 11 above)
Comment (TomShawn) [inline releases/release-4.0.11.md:38]:
```suggestion
    - 为 DBaaS 添加 server 信息的监控指标 [#9591](https://github.com/tikv/tikv/pull/9591)
```

File: releases/release-4.0.11.md (hunk 12 above)
Comment (TomShawn) [reply releases/release-4.0.11.md:7]: @alex-quan-001 I will update the date once the release date is determined.
```suggestion
发版日期：2021 年 2 月  日（待确定）
```

File: releases/release-4.0.11.md (hunk 11 above)
Comment (TomShawn) [reply releases/release-4.0.11.md]: Fixed in https://github.com/pingcap/docs-cn/pull/5458/commits/83b3704df63327c9ddcc75dbac057936f51eea7f.

File: releases/release-4.0.11.md [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+    - 修复成员健康的监控显示不正确的问题 [#3368](https://github.com/pingcap/pd/pull/3368)
+    - 禁止有副本的不正常 tombstone store 被清除 [#3352](https://github.com/pingcap/pd/pull/3352)
+    - 修复 store limit 无法持久化的问题 [#3403](https://github.com/pingcap/pd/pull/3403)
+    - 调整 `scatter range schedler` 的 limit 限制 [#3401](https://github.com/pingcap/pd/pull/3401)
+
++ TiFlash
+
+    - 修正 Decimal 类型的 `min`/`max` 计算结果错误的问题
+    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
+    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:139]:
```suggestion
    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题 [#1350](https://github.com/pingcap/tics/pull/1350)
    - 修正 Coprocessor 中错误的解析了 Decimal 常量的问题
    - 修正 Learner Read 过程中可能导致 crash 的问题
    - 修正 TiFlash 除 `0` 或 `NULL` 与 TiDB 行为不一致的问题
```

File: releases/release-4.0.11.md [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+    - 改进 Fast Tune 面板 [#9180](https://github.com/tikv/tikv/pull/9180)
+
++ PD
+
+    - Grafana dashboards 支持监控多个集群 [#3398](https://github.com/tikv/tikv/pull/3398)
+
++ TiFlash
+
+    - 优化 `date_format` 函数的性能
+    - 优化处理 ingest SST 时的内存开销
```
Comment (zanmato1984) [inline releases/release-4.0.11.md:53]:
```suggestion
    - 优化处理 ingest SST 时的内存开销
    - 优化 Batch Coprocessor 内部的重试逻辑以降低 Region Error 的出现概率
```

File: releases/release-4.0.11.md [hunk 15]
```diff
@@ -0,0 +1,169 @@
+---
+title: TiDB 4.0.11 Release Notes
+---
+
+# TiDB 4.0.11 Release Notes
+
+发版日期：2021 年 2 月  日（待确定）
```
Comment (TomShawn) [inline releases/release-4.0.11.md:7]:
```suggestion
发版日期：2021 年 2 月 26 日
```

File: releases/release-4.0.11.md [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+    - 支持 `uft8_unicode_ci` 和 `utf8mb4_unicode_ci` 排序规则 [#22558](https://github.com/pingcap/tidb/pull/22558)
+
++ TiKV
+
+    - 支持 `utf8mb4_unicode_ci` 排序规则 [#9577](https://github.com/tikv/tikv/pull/9577)
+    - 支持 `cast_year_as_time` 排序规则 [#9299](https://github.com/tikv/tikv/pull/9299)
+
++ TiFlash
+
+    - 增加排队处理 Coprocessor 任务的线程池以减小内存溢出几率，并增加配置项 `cop_pool_size` 和 `batch_cop_pool_size`，默认值为 `物理核数 * 2` [#1312](https://github.com/pingcap/tics/pull/1312)
```
Comment (qiancai) [inline releases/release-4.0.11.md:24]: “减小内存溢出几率”中的“减小”是否可以改成“降低”？

File: releases/release-4.0.11.md [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+    - 修复 `like` 函数使用排序规则时可能返回错误结果的问题 [#22531](https://github.com/pingcap/tidb/pull/22531)
+    - 修复 `least` 和 `greatest` 函数 `duration` 类型推导错误问题 [#22580](https://github.com/pingcap/tidb/pull/22580)
+    - 修复 `like` 函数处理 `_` 宽字符后加 `%` 出错的问题 [#22575](https://github.com/pingcap/tidb/pull/22575)
+    - 修复比较函数 `least` 和 `greatest` 类型推导错误的问题 [#22562](https://github.com/pingcap/tidb/pull/22562)
+    - 修复使用 `like` 函数处理 Unicode 字符串错误的问题 [#22529](https://github.com/pingcap/tidb/pull/22529)
+    - 修复点查请求无法取得 `@@tidb_snapshot` 变量中快照的问题 [#22527](https://github.com/pingcap/tidb/pull/22527)
+    - 修复生成多个 join 相关 hint 可能 panic 的问题 [#22518](https://github.com/pingcap/tidb/pull/22518)
+    - 修复转换字符串为 `BIT` 类型不准确的问题 [#22420](https://github.com/pingcap/tidb/pull/22420)
+    - 修复插入 `tidb_rowid` 列时出现的 `index out of range` 报错问题 [#22359](https://github.com/pingcap/tidb/pull/22359)
+    - 修复缓存计划被错误的使用问题 [#22353](https://github.com/pingcap/tidb/pull/22353)
```
Comment (qiancai) [inline releases/release-4.0.11.md:85]: “修复缓存计划被错误的使用问题” looks confusing, does it mean “修复缓存计划被错误使用的问题” or “修复缓存计划被错误地使用的问题”？

File: releases/release-4.0.11.md [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+
++ TiFlash
+
+    - 修正 Decimal 类型的 `min`/`max` 计算结果错误的问题
+    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
+    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题
+    - 修正 Coprocessor 中错误解析 Decimal 常量的问题
+    - 修正 Learner Read 过程中可能导致 crash 的问题
+    - 修正 TiFlash 中除 `0` 或 `NULL` 的行为与 TiDB 不一致的问题
+
```
Comment (qiancai) [inline releases/release-4.0.11.md:144]: line 138 to line 143, the verb "修正“ is used, while in all other notes in this document, "修复“ is used. Suggest using "修复“ consistently in this document.

File: releases/release-4.0.11.md [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
++ PD
+
+    - 修复成员健康的监控显示不正确的问题 [#3368](https://github.com/pingcap/pd/pull/3368)
+    - 禁止有副本的不正常 tombstone store 被清除 [#3352](https://github.com/pingcap/pd/pull/3352)
+    - 修复 store limit 无法持久化的问题 [#3403](https://github.com/pingcap/pd/pull/3403)
+    - 调整 `scatter range schedler` 的 limit 限制 [#3401](https://github.com/pingcap/pd/pull/3401)
+
++ TiFlash
+
+    - 修正 Decimal 类型的 `min`/`max` 计算结果错误的问题
```
Comment (qiancai) [inline releases/release-4.0.11.md:138]: No PR link is provided after this item.

File: releases/release-4.0.11.md [hunk 20]
```diff
@@ -0,0 +1,169 @@
...
+    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
+    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题
+    - 修正 Coprocessor 中错误解析 Decimal 常量的问题
+    - 修正 Learner Read 过程中可能导致 crash 的问题
+    - 修正 TiFlash 中除 `0` 或 `NULL` 的行为与 TiDB 不一致的问题
```
Comment (qiancai) [inline releases/release-4.0.11.md:143]: line 140 to line 143, No PR link is provided after these items.

Comment (TomShawn) [reply releases/release-4.0.11.md:143]: TiFlash is not open-sourced, so TiFlash notes have no PRs appended.

Comment (qiancai) [inline releases/release-4.0.11.md:143]: In "修正 TiFlash 中除 `0` 或 `NULL` 的行为与 TiDB 不一致的问题",  is "除 `0` 或 `NULL` 的行为" correct? Would you please double check?

Comment (qiancai) [reply releases/release-4.0.11.md:143]: > TiFlash is not open-sourced, so TiFlash notes have no PRs appended.

OK. Thanks for the explanation. Please ignore the comment.

File: releases/release-4.0.11.md (hunk 17 above)
Comment (TomShawn) [reply releases/release-4.0.11.md:85]: The English version is "Fix a bug that the cached plan is incorrectly used". I think it should be “修复缓存计划被错误地使用的问题”.

File: releases/release-4.0.11.md (hunk 20 above)
Comment (TomShawn) [reply releases/release-4.0.11.md:143]: @zanmato1984 Would you please take a look? Thanks!

Comment (TomShawn) [reply releases/release-4.0.11.md:143]: As confirmed with @zanmato1984, it should be "除以 `0` 或 `NULL`". However, "除 `0` 或 `NULL`" is also commonly used among programmers.

File: releases/release-4.0.11.md (hunk 18 above)
Comment (TomShawn) [reply releases/release-4.0.11.md:144]: Fixed

File: releases/release-4.0.11.md (hunk 16 above)
Comment (TomShawn) [reply releases/release-4.0.11.md]: Yes. Changed to “降低”.

Comment (qiancai) [reply releases/release-4.0.11.md]: LGTM

**Comment by TomShawn** [comment]
@qiancai All comments are addressed. PTAL again, thanks!

File: releases/release-4.0.11.md (hunk 17 above)
Comment (qiancai) [reply releases/release-4.0.11.md]: LGTM

File: releases/release-4.0.11.md (hunk 18 above)
Comment (qiancai) [reply releases/release-4.0.11.md:144]: LGMT

File: releases/release-4.0.11.md (hunk 20 above)
Comment (qiancai) [reply releases/release-4.0.11.md]: LGMT

**Comment by qiancai** [comment]
LGMT

**Review by TomShawn — APPROVED** [review]

**Comment by scsldb** [comment]
LGTM

**Pull request merged**